base/applications/mscutils/eventvwr/evtdetctl.c

```diff
@@ -878,20 +878,15 @@ EventDetailsCtrl(HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     return (INT_PTR)TRUE;
 
                 case IDC_BYTESRADIO:
-                    if (pData->EventLogFilter)
-                    {
-                        pData->bDisplayWords = FALSE;
-                        DisplayEventData(hDlg, pData->bDisplayWords);
-                    }
-                    return (INT_PTR)TRUE;
-
                 case IDC_WORDRADIO:
+                {
```

Comment:
Personally I am no fan of introducing such curly-braces for switch-cases unless there is some variable declaration within that, which is not the case here. But I guess this is a matter of personal preference.
Same applies for the other places where you added those.

Reply:
I put brackets also for case blocks when they start to become large (>= 4 lines) as this allows to fold them in any code editor.